src/index.html

```diff
@@ -11,7 +11,164 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form class="Registration"
+     action="linthtml"
+     method="post"
+     autocomplete="off">
+      <fieldset class="section pinf">
+        <legend>Personal information</legend>
+
+        <label class="form surname">
```

Comment:
```suggestion
        <label class="form-field">
```
consider more meaningful name. surname looks like useless in this case